src/program.js

```diff
@@ -123,11 +125,27 @@ export class Program {
   }
 }
 
+// A global variable generated by DefinePlugin, generated in webpack.config.js
+declare var WEBEXT_BUILD_ENV: string;
 
-export function defaultVersionGetter(absolutePackageDir: string): string {
-  let packageData: any = readFileSync(
-    path.join(absolutePackageDir, 'package.json'));
-  return JSON.parse(packageData).version;
+//A defintion of type of argument for defaultVersionGetter
```

Comment:
Just for future note: comments typically begin with a space elsewhere in the codebase, like `// A definition...`